source/quickstart/osx.rst

```diff
@@ -6,31 +6,213 @@ Quickstart for Mac OSX
 
 .. default-domain:: minio
 
-This page deploys MinIO on Mac OSX for use with early development
-and evaluation environments.
+.. |OS| replace:: MacOS
 
-For instructions on deploying to production environments,
-see :ref:`deploy-minio-distributed`.
+This page deploys MinIO on |OS| for use with early development and evaluation environments.
+
+For instructions on deploying to production environments, see :ref:`deploy-minio-distributed`.
+
+The procedure described here deploys MinIO in :guilabel:`Standlone Mode`` on a directory on the |OS| harddrive.
+A standalone MinIO deployment consists of a single MinIO server process with a single drive or storage volue (what we call "filesystem mode").
+The MinIO server provides an S3 access layer to the drive or volume.
+
+Objects store to the volume as-is with no :ref:`erasure coding <minio-erasure-coding>`.
 
 Prerequisites
 -------------
 
-ToDo
+- Read, write, and execute permissions for the user's home directory
+- Familiarity with using the Terminal
+
+.. admonition:: Network File System Volumes Break Consistency Guarantees
+   :class: note
+
+   MinIO's strict **read-after-write** and **list-after-write** consistency model requires local disk filesystems (``xfs``, ``ext4``, etc.).
+
+   MinIO cannot provide consistency guarantees if the underlying storage volumes are NFS or a similar network-attached storage volume. 
+
+   For deployments that *require* using network-attached storage, use NFSv4 for best results.
 
 Procedure
 ---------
 
-#. Step 1
+Install the MinIO Server
```

Comment:
So you can still number these, but you might have to escape some characters. I _think_:

```
1\) Install the MinIO Server
~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```


Might work.

Reply:
I'm trying to avoid having to manually number things.
It's just a maintenance headache when the steps required inevitably change.